.devcontainer/Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG DEVCONTAINER_VERSION=bullseye
+FROM mcr.microsoft.com/devcontainers/base:${DEVCONTAINER_VERSION}
```

Comment:
why not go for a python devcontainer?
mcr.microsoft.com/vscode/devcontainers/python:<version>